### PR TITLE
Release 1.0.0.0

### PR DESCRIPTION
**Pull request by dszmigielski**
### Breaking changes

- Replace `SIGNALFX_RUNTIME_METRICS_ENABLED` setting with `SIGNALFX_METRICS_{0}_ENABLED`,
that allows to enable specific metrics groups (See [advanced-config](advanced-config.md)).
- Rename `SIGNALFX_TRACE_METRICS_ENABLED` to `SIGNALFX_METRICS_Traces_ENABLED` to
preserve consistent naming convention.

### Enhancements

- Add support for .NET 7.0.
- Add `SIGNALFX_TRACE_BUFFER_SIZE` setting to configure trace exporter buffer size.
- Add `SIGNALFX_METRICS_{0}_ENABLED` to enable specific metrics group.
- Change metric structure of the runtime metrics exported to align with the OpenTelemetry
implementation from the [Runtime](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/bc947a00c3f859cc436f050e81172fc1f8bc09d7/src/OpenTelemetry.Instrumentation.Runtime) package.
- Change metric structure of the process metrics exported to align with the OpenTelemetry
implementation from the [Process](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/926386e68c9066e8032853e8309abdf4088d8dca/src/OpenTelemetry.Instrumentation.Process) package.

### CODE REVIEW

**Review by theletterf — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -13,15 +13,25 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
...
 
-### Bugfixes
-
 ### Enhancements
 
 - Add `SIGNALFX_TRACE_BUFFER_SIZE` setting to configure trace exporter buffer size.
 - Add `SIGNALFX_METRICS_{0}_ENABLED` to enable specific metrics group.
+- Change metric structure of the runtime metrics exported to align with the OpenTelemetry
+implementation from the [Runtime](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/bc947a00c3f859cc436f050e81172fc1f8bc09d7/src/OpenTelemetry.Instrumentation.Runtime) package.
+- Change metric structure of the process metrics exported to align with the OpenTelemetry
```
Comment (pellared) [inline docs/CHANGELOG.md:33]: I think we added support for .NET 7.0.

Comment (Kielek) [reply docs/CHANGELOG.md:33]: 0.2.9 is working with . NET 7. We have added mostly tests (and partially extended set of instrumented libraries).

Comment (pellared) [reply docs/CHANGELOG.md:33]: Still I think it is worth to call out that we added support for .NET 7 as some of the fixed stuff was for the BCL like https://github.com/signalfx/signalfx-dotnet-tracing/pull/663 https://github.com/signalfx/signalfx-dotnet-tracing/pull/665 https://github.com/signalfx/signalfx-dotnet-tracing/pull/666

Comment (dszmigielski) [reply docs/CHANGELOG.md:34]: Added

**Review by Kielek — APPROVED** [review]

**Pull request merged**